### DISCUSSION TITLE
Release v1.0.15

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#5461] [Polls]
https://github.com/JanssenProject/jans/milestone/18

**Comment by manojs1978** [comment]
https://github.com/JanssenProject/jans/issues/5458
need to fix this as tui is crashing

**Comment by yuriyz** [comment]
@shekhar16 cache refresh prevents FullRebuild from running. Can you check ?
https://jenkins.jans.io/jenkins/job/FullRebuild/1912/

**Reply by yuriyz** [reply]
It is solved. AS tests are passed successfully.

**Comment by manojs1978** [comment]
pre release test Result


------------------------------------------------------------------------------------------------------------------------------
|    |Opendj | Mysql |Pgsql |.well-known config| device-flow |simple_password_auth| Agama Project deployment | comment|
| ---|-------|-------|------|------------------|-------------|---------------------|-------------------------|---------|
|suse|       | Pass  | Pass | Pass             | Pass          |         Pass        | Pass                     | Pass  | 
|RHEL|  | Pass  | Pass | Pass             | Pass          |         Pass        | Pass                     | Pass |
|ubuntu20|Pass  | Pass  | Pass | Pass             | Pass          |         Pass        | Pass                     | Pass|
|ubuntu22|Pass  | Pass  | Pass | Pass             | Pass          |         Pass        | Pass                     | Pass| 
------------------------------------------------------------------------------------------------------------------------------

Note: i am always facing issue on aws VM when installing jans/flex with opendj but others are not facing same  issue

**Comment by moabu** [comment]
https://github.com/JanssenProject/jans/releases/tag/v1.0.15
